### ISSUE TITLE
null dereference on parsing malform html

### ISSUE THREAD

**Issue by kcwu** [dafyddcrosby/sw3m#17]
How to reproduce
```
$ cat crash.html
<ol type 0=>
$ sw3m crash.html
Segmentation fault
```
gdb call stack
```
(gdb) bt
#0  __strcmp_sse2_unaligned () at ../sysdeps/x86_64/multiarch/strcmp-sse2-unaligned.S:30
#1  0x000000000046fbca in parse_tag (s=0x7fffffffd0c0, internal=0) at parsetagx.c:225
#2  0x000000000042a43a in HTMLlineproc0 (line=0x7b4c6c "\n", h_env=0x7fffffffd1a0, internal=0) at file.c:6215
#3  0x000000000042c764 in loadHTMLstream (f=0x7fffffffd770, newBuf=0x7e9d80, src=0x0, internal=0) at file.c:6937
#4  0x000000000042b2c0 in loadHTMLBuffer (f=0x7fffffffd770, newBuf=0x7e9d80) at file.c:6524
#5  0x00000000004160c5 in loadSomething (f=0x7fffffffd770, path=0x7b5900 "/home/kcwu/fuzz/targets/w3m/run/crash.html", loadproc=0x42b1da <loadHTMLBuffer>, defaultbuf=0x0) at file.c:221
#6  0x000000000041bed9 in loadGeneralFile (path=0x7fffffffdeba "crash.html", current=0x0, referer=0xffffffffffffffff <error: Cannot access memory at address 0xffffffffffffffff>, flag=0, request=0x0)
    at file.c:2188
#7  0x0000000000406c10 in main (argc=2, argv=0x7fffffffda78, envp=0x7fffffffda90) at main.c:848

(gdb) up
#1  0x000000000046fbca in parse_tag (s=0x7fffffffd0c0, internal=0) at parsetagx.c:225
225                    strcmp("hidden",tag->value[j]) == 0) {
(gdb) p tag->value[j]
$1 = 0x0
(gdb) p j
$2 = 1
```

this is found by afl-fuzz

**Comment by dafyddcrosby** [comment]
fixed in master

**Comment by kcwu** [comment]
Just to keep reference, this is fixed by 82e2cceccc400966a607690b0f1a4b91894eebd4

**Comment by kcwu** [comment]
cc @tats

**Comment by tats** [comment]
> fixed by 82e2cce

In Debian, already fixed by https://github.com/tats/w3m/commit/608828ed7e3c47560b9653dde5141f27c860449a

**Comment by kcwu** [comment]
thanks
